.travis.yml

```diff
@@ -35,19 +35,18 @@ matrix:
     - php: 5.6
       env: SYMFONY_VERSION=2.8.*@dev
     - php: 5.6
-      env: SYMFONY_VERSION="3.0.x-dev as 2.8"
+      env: SYMFONY_VERSION="3.0.x-dev" DEPENDENCIES="dev" COMPOSER_FLAGS="--prefer-stable"
```

Comment:
Why not just `3.0.*@dev`?

Event better, set minimum-stability to dev and prefer-stable to true directly on `composer.json` file and remove `@dev` annotations.